### ISSUE TITLE
Transparent desktop: rearrange settings

### ISSUE THREAD

**Issue by Vladimir-csp** [IgnorantGuru/spacefm#600]
Hi!
This is more a cosmetic issue, but it still bugs me in terms of interface logic. Desktop background settings should be rearranged into this hierarchy:

Desktop background mode:
- image
  - image selection
- solid color
  - color selection
- transparent

**Comment by IgnorantGuru** [comment]
The Prefs dialog is a mess, SpaceFM really needs a new settings dialog.  The ui file might be able to be edited to rearrange some of the order, but I think colors apply even when an image is shown (for background colors, etc).

**Comment by Vladimir-csp** [comment]
Yes, your're right. Potentially for #408 also.

**Comment by Vladimir-csp** [comment]
How does transparent mode work? If it is RGBA color with 0 Alpha channel, then it could be merged into color selection and allow partially transparent desktop

**Comment by IgnorantGuru** [comment]
Beats me - I've never used it.  You would have to ask @BwackNinja or one of the more experienced transparency people that one.

**Comment by BwackNinja** [comment]
That's correct. RGBA mode is just 0 value for the alpha channel. While it's certainly possible to merge it with the color selection, I think it makes sense to keep it as a separate use-case where the wallpaper is handled by another program. It also only works with compositing window managers, and giving a range of alpha values selectable could be more confusing. If you or others want the possibility of a partially transparent desktop, we can look into that.
